CommonLibSF/include/RE/T/TESObjectREFR.h

```diff
@@ -285,14 +285,15 @@ namespace RE
 
 		// NiPointer<TESObjectREFR>
 		[[nodiscard]] TESObjectREFR* GetAttachedSpaceship();
-		[[nodiscard]] TESWorldSpace* GetParentWorldSpace();
-
 		[[nodiscard]] TESBoundObject*       GetBaseObject() { return data.objectReference; }
 		[[nodiscard]] const TESBoundObject* GetBaseObject() const { return data.objectReference; };
+		[[nodiscard]] BGSLocation*   GetCurrentLocation();
+		[[nodiscard]] TESWorldSpace* GetParentWorldSpace();
 		[[nodiscard]] constexpr NiPoint3A   GetPosition() const noexcept { return data.location; }
 		[[nodiscard]] constexpr float       GetPositionX() const noexcept { return data.location.x; }
 		[[nodiscard]] constexpr float       GetPositionY() const noexcept { return data.location.y; }
 		[[nodiscard]] constexpr float       GetPositionZ() const noexcept { return data.location.z; }
+		[[nodiscard]] TESObjectREFR* GetSpaceshipPilot();
```

Comment:
Should return `Actor` instead of `TESObjectREFR`?  One of the calling function does a check to see if the formtype is `NPC`.

Reply:
Yes, sorry, form type is 75. Didn't check for this function.(BGSLocation is confirmed to be locatition type).
Will update shortly.